test/utils/RegexPatternsSpec.scala

```diff
@@ -45,6 +45,28 @@ class RegexPatternsSpec extends SpecBase {
     }
   }
 
+  "UCR Regex" should {
```

Comment:
Better to maintain the order of the tests same as the vals are in the object. So better to move these tests after "Payment Regex"